packages/bot/src/assets/sass/scratch/flyout.scss

```diff
@@ -52,26 +81,51 @@
             display: flex;
             justify-content: flex-end;
         }
-        &-add {
-            padding: 5px 20px;
-            border: none;
-            font-size: 16px;
-            color: $COLOR_WHITE;
-            border-radius: 5px;
-            font-weight: bold;
-            margin-left: $default-margin;
-            background-color: $COLOR_ORANGE;
-            cursor: pointer;
-        }
         &-info {
             cursor: pointer;
+            color: $COLOR_RED;
+        }
+        &-description {
+            font-size: 0.8em;
+            margin-bottom: 1em;
+        }
+    }
+    &__image {
+        width: 100%;
+        height: auto;
+        border-radius: 0.5em;
+    }
+    &__video {
+        width: 100%;
+        height: 20vh;
+        border-radius: 0.5em;
+    }
+    &__help {
+        padding: 0;
+
+        &-header {
+            padding: 1.2em 0;
             display: flex;
+            background-color: $COLOR_LIGHT_BLACK_4_SOLID;
+        }
+        &-content {
+            padding: 1.5em;
+            overflow-y: auto;
+            height: calc(100vh - 300px);
 
-            > .icon-info {
-                height: 20px;
-                width: 20px;
-                align-self: center;
+            .flyout__item {
```

Comment:
You can create a variable containing the flyout's class name in the `.flyout` scope, i.e. `$flyout = &;` and then reference it here using `#{$flyout}__item`. You can check `toolbox.scss` as an example.